UI/index.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8">
+    <title>Title</title>
+    <link href="style.css" rel="stylesheet">
+</head>
+<body>
+<header>
+    <div class="header-container">
+        <img src="logo.png" class="img-logo" alt="logo">
+        <img src="logo-small.png" id="logo-small" alt="logo">
+        <div class="input-block flex-row">
+            <input type="text" class="search-input" placeholder="Search">
+            <a href="#" style="text-align: right; width: 100%; margin: 0; padding: 0; font-size: 14px;">Advanced Search</a>
```

Comment:
There are a lot of inline styles. It's bad practice to use them, try to avoid it.